dev/com.ibm.ws.security.audit.file/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -39,13 +39,15 @@ encrypt.desc=Indicates whether the audit records in the audit file log will be e
 encryptAlias.name=Alias name of certificate for encrypting
 encryptAlias.desc=Alias name of the certificate used to encrypt the audit records.
```

Comment:
Suggested edit:
Alias name of the certificate that is used to encrypt the audit records.

Reply:
Message updated.  Please take a look.